builtin/logical/database/path_role_create.go

```diff
@@ -0,0 +1,93 @@
+package database
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/vault/helper/strutil"
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathRoleCreate(b *databaseBackend) *framework.Path {
```

Comment:
I realize this is a holdover from the past, but can we name this (and the file) something more descriptive, e.g. `pathCreds`?